tests/dotnet/UnitTests/ResourcePrefixTest.cs

```diff
@@ -0,0 +1,138 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework;
+
+namespace Xamarin.Tests {
+	[TestFixture]
+	public class ResourcePrefixTest : TestBaseClass {
+		[Test]
+		public void ResourcePrefix_DefaultValues ()
+		{
+			// Arrange
+			var platforms = Configuration.GetAllPlatforms ();
```

Comment:
Use `GetIncludedPlatforms` instead of `GetAllPlatforms` in all cases in this file.